cmd/login.go

```diff
@@ -7,9 +7,9 @@ import (
 	"os"
 
 	magellan "github.com/OpenCHAMI/magellan/internal"
-	"github.com/OpenCHAMI/magellan/internal/log"
+	"github.com/OpenCHAMI/magellan/internal/util"
```

Comment:
Creating a util package is usually a sign that you've made some incorrect assumptions about composability/modularity.

https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Is there a missing proper library here, perhaps called auth?  Could it be made more general to allow other auth methods to be used in the future?

Reply:
Nope. I believe that can be split into several other packages, but I just made it `util` for convenience.